### PR TITLE
 feat: [FC-10] Message status when send message

### PR DESCRIPTION
**Pull request by huydosgtech**

### CODE REVIEW

File: src/chat/ChatScreen.tsx [hunk 1]
```diff
@@ -177,6 +209,54 @@ export const ChatScreen: React.FC<ChatScreenProps> = ({
...
+      right: { backgroundColor: 'transparent' },
+    };
+
+    return (
+      <Bubble
+        {...bubble}
+        renderCustomView={() =>
+          bubble.currentMessage && (
+            <View>
+              <CustomBubble
```
Comment (huydosgtech) [inline src/chat/ChatScreen.tsx:246]: this component come from [FC-5]

**Comment by danhmaisgt** [comment]
Can you add a demo video or something please? It will be easy for me to understand your implementation.

File: src/chat/ChatScreen.tsx [hunk 2]
```diff
@@ -137,6 +148,26 @@ export const ChatScreen: React.FC<ChatScreenProps> = ({
...
+        setUserUnreadMessage(hasUnreadMessages);
+      }
+    );
+
+    return () => {
+      if (userConversation) {
+        userConversation();
+      }
+    };
+  }, [firebaseInstance, partners]);
```
Comment (loc-nguyenthien) [inline src/chat/ChatScreen.tsx:169]: firebaseInstance is a ref, so we don't need it here

**Review by loc-nguyenthien — CHANGES_REQUESTED** [review]

**Comment by loc-nguyenthien** [comment]
Added video demo for chat status

https://github.com/user-attachments/assets/f83ab770-de0b-4cb7-b10c-0401c76c27a1

File: src/services/firebase/firestore.ts [hunk 3]
```diff
@@ -381,11 +381,23 @@ export class FirestoreServices {
...
+            unRead = conversationData.unRead;
+          }
+        }
+
+        unRead[userId] = messageId;
+        transaction.set(conversationRef, { unRead }, { merge: true });
+        transaction.set(userConversationRef, { unRead: 0 }, { merge: true });
+      });
+
+      console.log('Unread message ID updated successfully');
```
Comment (loc-nguyenthien) [inline src/services/firebase/firestore.ts:400]: Remove log

**Review by loc-nguyenthien — APPROVED** [review]

**Pull request merged**